overnight-battery-loan/form.tex

```diff
@@ -22,26 +24,38 @@
 \begin{center}
 \begin{table}[h]
 \def\arraystretch{1.5}
-\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|Y|Y|Y|Y|}
-    \textbf{TLA} & \textbf{Charger Part Number} & \textbf{Battery Bag Part Number} & \textbf{Battery 1 Part Number} & \textbf {Battery 2 Part Number} & \textbf{Charger Returned?} & \textbf{Battery Bag Returned?} & \textbf{Battery 1 Returned?} & \textbf{Battery 2 Returned?} \\
+\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|}
+    \textbf{TLA} &
+    \textbf{Charger Part Number} &
+    \textbf{Battery Bag Part Number} &
+    \textbf{Battery 1 Part Number} &
+    \textbf{Battery 2 Part Number} \\
 \hline
 \large
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
-& & & & & & & & \\ \hline
+& & & & \\ \arrayrulecolor{gray} \hline \arrayrulecolor{black}
```

Comment:
Looking at this rendered out on my screen, it's very difficult to see the distinction between black and grey on a line as thin as this. It might be nicer to use a thicker line for the border between row groups.

Reply:
Hrm, for me at the zoom level where it's at real size it seems fine. I'll try to print a copy tomorrow and see how it looks in person.